docs/Development-environment-on-Ubuntu-12.04.md

```diff
@@ -6,81 +6,78 @@ These are the main steps to install the OpenQuake Platform on Ubuntu 12.04 LTS u
 sudo apt-get install build-essential python-dev python-mock python-imaging python-virtualenv git postgresql-9.1 postgresql-server-dev-9.1 postgresql-contrib-9.1 postgresql-9.1-postgis openjdk-6-jre libxml2 libxml2-dev libxslt1-dev libxslt1.1 libblas-dev liblapack-dev curl wget xmlstarlet imagemagick gfortran python-nose libgeos-dev
 ```
 
-## Install optional dependencies to be able to use nrml validator from risklib package
+## Change the PostgreSQL configuration
+
+On top of `/etc/postgresql/9.1/main/pg_hba.conf` add:
 
 ```
-#
-# ---- if your virtual environment HAS ACCESS to system-wide packages ----
-#
+local   all             all                                     trust
+```
 
-sudo apt-get install python-software-properties
-sudo add-apt-repository ppa:openquake/ppa
-sudo apt-get update
-sudo apt-get install python-decorator python-h5py python-psutil python-concurrent.futures python-oq-hazardlib python-oq-engine
+Then restart PostgreSQL
+
+## Clone the GitHub repo
+```bash
+git clone http://github.com/gem/oq-platform.git
+```
 
-# OR install it from github using pip
+## Create the virtualenv
+```bash
+cd ~/oq-platform
+virtualenv --system-site-packages platform-env
+. platform-env/bin/activate
+```
+
+## Install software dependencies
+### Install OpenQuake Engine and Hazardlib
+
+#### Using a virtualenv with access to the global site-packages (default)
+
+```bash
 sudo apt-get install python-software-properties
 sudo add-apt-repository ppa:openquake/ppa
 sudo apt-get update
 sudo apt-get install python-decorator python-h5py python-psutil python-concurrent.futures
-# (into virtualenv)
+
 pip install 'http://github.com/gem/oq-hazardlib/tarball/master'
 pip install 'http://github.com/gem/oq-engine/tarball/master'
 
-# OR download 'oq-hazardlib' and 'oq-engine' manually from github and make available via PYTHONPATH
-# before run any python applications
+```
 
-# ---- else if your virtual environment HASN'T ACCESS to system-wide packages ----
+#### Using a virtualenv without access to the global site-packages
 
+```bash
 # you can install all required packages using pip:
 pip install futures
 pip install decorator
 pip install psutil
 pip install Cython
 pip install h5py
-pip install selenium
```

Comment:
@nastasi-oq I removed selenium from here because it is not an engine dep. But maybe we should add a section about installing everything is needed to run selenium.